packages/kotti-ui/source/kotti-field-file-upload/utils/common.ts

```diff
@@ -0,0 +1,57 @@
+import dayjs from 'dayjs'
+import filesize from 'filesize'
+
+import { ISO8601_SECONDS } from '../constants'
+import { KottiFieldFileUpload } from '../types'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const nanoid = require('nanoid')
+
+export const formatFileSize = (fileSize: number): string =>
+	filesize(fileSize, { round: 1, spacer: '' })
```

Comment:
```suggestion
	filesize(fileSize, { base: 2, round: 1, spacer: '' })
```

Unless necessary, I'd also remove the spacer override to ease readability